src/source/XplicitNgine/XplicitNgine.cpp

```diff
@@ -83,7 +83,9 @@ void XplicitNgine::createBody(PartInstance* partInstance)
 {
 	// calculate collisions
 	//dSpaceCollide (physSpace,0,&collisionCallback);
-    
+
+	Vector3 partSize = partInstance->getSize();

```

Comment:
Ah right, getSize returns by "value", not a pointer. This would make it heaps more efficient if I am not mistaken.